tools/jcw-gen/App.cs

```diff
@@ -36,6 +37,9 @@ public static int Main (string [] args)
 				{ "o=",
 				  "{DIRECTORY} to write Java source code to.",
 				  v => outputPath = v },
+				{ "codegen-target=",
+				  "STYLE of Java Callable Wrappers to generate",
+				  (JavaPeerStyle? v) => style = v.HasValue ? v.Value : style },
```

Comment:
Should the switch here also be named `--java-peer-style=`?

Reply:
The intent was to be consistent with `generator`, so you would use `generator --codegen-target=JavaInterop1 … ; jcw-gen --codegen-target=JavaInterop1 …`.

That might not actually be useful, especially considering that MSBuild tasks should (eventually) be provided, not command-line tools…